demos/serverless/src/handlers.js

```diff
@@ -3,8 +3,8 @@
 
 const AWS = require('aws-sdk');
 const fs = require('fs');
-const { v4: uuidv4 } = require('uuid');
-const { metricScope } = require('aws-embedded-metrics');
+const {v4: uuidv4} = require('uuid');
```

Comment:
Looks like local formatting changes, sorry too bad we dont have linter in JS SDK demo I think, will track it but can you keep it as it is.

Reply:
okay I will revert the format change.